src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Mapping/ToViewModel/ManageEvidenceNotesViewModelMapTests.cs

```diff
@@ -195,13 +194,98 @@ public void Map_GivenSourceWithCurrentDate_ComplianceYearListShouldBeMapped()
+        {
+            //arrange
+            var aatfs = TestFixture.CreateMany<AatfData>().ToList();
+            var aatfId = TestFixture.Create<Guid>();
+            var currentDate = new DateTime(2021, 01, 31);
+            var complianceYear = currentDate.Year;
+
+            var source = new ManageEvidenceNoteTransfer(organisationId, aatfId, aatfData, aatfs, null, null, null, complianceYear, currentDate);
+
+            A.CallTo(() => aatfEvidenceHelper.AatfCanEditCreateNotes(A<List<AatfData>>.That.IsSameAs(aatfs), aatfId, complianceYear)).Returns(canCreateEdit);
+
+            //act
+            var result = map.Map(source);
+
+            //assert
+            result.CanCreateEdit.Should().Be(canCreateEdit);
+        }
+
+        public static IEnumerable<object[]> ClosedComplianceDates =>
+            new List<object[]>
+            {
+                new object[] { 2020, new DateTime(2021, 02, 1) },
+                new object[] { 2019, new DateTime(2021, 02, 1) },
+                new object[] { 2023, new DateTime(2021, 02, 1) }
+            };
+
+        [Theory]
+        [MemberData(nameof(ClosedComplianceDates))]
+        public void Map_GivenAatfCreateEvidenceNotesAndOutOfComplianceYear_CanCreateEditShouldBeSet(int complianceYear, DateTime currentDate)
+        {
+            //arrange
+            var aatfs = TestFixture.CreateMany<AatfData>().ToList();
+            var aatfId = TestFixture.Create<Guid>();
+
+            var source = new ManageEvidenceNoteTransfer(organisationId, aatfId, aatfData, aatfs, null, null, null, complianceYear, currentDate);
+
+            A.CallTo(() => aatfEvidenceHelper.AatfCanEditCreateNotes(A<List<AatfData>>.That.IsSameAs(aatfs), aatfId, complianceYear)).Returns(true);
+
+            //act
+            var result = map.Map(source);
+
+            //assert
+            result.CanCreateEdit.Should().Be(false);
+        }
+
+        [Theory]
+        [MemberData(nameof(ClosedComplianceDates))]
+        public void Map_GivenOutOfComplianceYear_ComplianceYearClosedShouldBeTrue(int complianceYear, DateTime currentDate)
+        {
+            //arrange
+            var source = new ManageEvidenceNoteTransfer(organisationId, aatfId, aatfData, new List<AatfData>(), null, null, null, complianceYear, currentDate);
+
+            //act
+            var result = map.Map(source);
+
+            //assert
+            result.ComplianceYearClosed.Should().BeTrue();
+        }
+
+        public static IEnumerable<object[]> OpenComplianceDates =>
+            new List<object[]>
+            {
+                new object[] { 2020, new DateTime(2021, 01, 31) },
+                new object[] { 2020, new DateTime(2020, 12, 31) },
+                new object[] { 2020, new DateTime(2020, 1, 1) },
+            };
+
+        [Theory]
+        [MemberData(nameof(OpenComplianceDates))]
+        public void Map_GivenInComplianceYear_ComplianceYearClosedShouldBeFalse(int complianceYear, DateTime currentDate)
+        {
```

Comment:
can you add a test for Map_GivenAatfCreateEvidenceNotesAndInfComplianceYear and Can Create Edit should be set?